src/main/java/seedu/address/logic/commands/ExportEventXmlCommand.java

```diff
@@ -0,0 +1,139 @@
+        }
+
+        return new CommandResult(MESSAGE_EXPORT_EVENT_SUCCESS);
+    }
+
+    /**
+     * Helper method to do the file creation and data writing
+     * @param model model to take the entire list
+     * @param event the specified event given on index
+     * @throws Exception - any given internal errors are thrown here
+     */
+    private void createEventXml (Model model, Event event) throws Exception {
+        //setting up the document builders
+        DocumentBuilderFactory dbFactory = DocumentBuilderFactory.newInstance();
+        DocumentBuilder dBuilder = dbFactory.newDocumentBuilder();
+        Document doc = dBuilder.newDocument();
+
+        //root element - currently tag as event
+        Element rootElement = doc.createElement(exportTypeE);
+        doc.appendChild(rootElement);
+
+        //root attributes to this particular event listing the details
+        Attr eventNameAttr = doc.createAttribute("Name");
+        eventNameAttr.setValue(event.getName().toString());
+        rootElement.setAttributeNode(eventNameAttr);
+
+        Attr eventStartDateAttr = doc.createAttribute("Start Date");
+        eventStartDateAttr.setValue(event.getStartDate().toString());
+        rootElement.setAttributeNode(eventStartDateAttr);
+
+        Attr eventStartTimeAttr = doc.createAttribute("Start Time");
+        eventStartTimeAttr.setValue(event.getStartTime().toString());
+        rootElement.setAttributeNode(eventStartTimeAttr);
+
+        Attr eventEndDateAttr = doc.createAttribute("End Date");
+        eventEndDateAttr.setValue(event.getEndDate().toString());
+        rootElement.setAttributeNode(eventEndDateAttr);
+
+        Attr eventEndTimeAttr = doc.createAttribute("End Time");
+        eventEndTimeAttr.setValue(event.getEndTime().toString());
+        rootElement.setAttributeNode(eventEndTimeAttr);
+
+        Attr eventLocationAttr = doc.createAttribute("Location");
+        eventLocationAttr.setValue(event.getLocation().toString());
+        rootElement.setAttributeNode(eventLocationAttr);
+
+        Attr eventDescriptionAttr = doc.createAttribute("Description");
+        eventDescriptionAttr.setValue(event.getDescription().toString());
+        rootElement.setAttributeNode(eventDescriptionAttr);
+
+        Attr eventIdAttr = doc.createAttribute("Event ID");
+        eventIdAttr.setValue(event.getEventId().toString());
+        rootElement.setAttributeNode(eventIdAttr);
+
+        //elements 1 level below root - this is used to store volunteers for the event
+
+        // Setting up transformer
+        TransformerFactory transformerFactory = TransformerFactory.newInstance();
+        Transformer transformer = transformerFactory.newTransformer();
+        DOMSource source = new DOMSource(doc);
+
+        // Setting up file path
+        File output = new File(System.getProperty("user.home") + "/Desktop/"
+                + event.getName().toString() + ".xml");
```

Comment:
Suggestion to include the event ID here as well, such that the export file name takes the format `<EVENTNAME>_<EVENTID>.xml`